src/main/java/com/rachelbock/resources/CommentsResource.java

```diff
@@ -0,0 +1,135 @@
+package com.rachelbock.resources;
+
+import com.rachelbock.data.Comment;
+import com.rachelbock.db.ConnectionPool;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.sql.*;
+import java.time.Instant;
+import java.util.ArrayList;
+
+import java.util.List;
+
+/**
+ * Class to set up the database connection for comments
+ */
+@Path("/climbs/{climb_id}/comments")
+@Produces(MediaType.APPLICATION_JSON)
+@Consumes(MediaType.APPLICATION_JSON)
+public class CommentsResource {
+
+    public static final String GET_COMMENTS_QUERY = "SELECT * FROM comments INNER JOIN climbs \n" +
+            "ON comments.climb_id = climbs.climb_id \n" +
+            "WHERE climbs.climb_id = ? ORDER BY date_text DESC";
+
+
+    /**
+     * Retrieves comments based on climb id. Returns an empty list if there are no comments for that climb.
+     * @param climb_id - the id of the climb that the comments are for.
+     * @return - ArrayList of comments
+     */
+    @GET
+    public List<Comment> getCommentsForClimb(@PathParam("climb_id") int climb_id){
+        List<Comment> comments = new ArrayList<>();
+
+        try(Connection conn = ConnectionPool.getConnection()){
+
+            PreparedStatement stmt = conn.prepareStatement(GET_COMMENTS_QUERY);
+            stmt.setInt(1, climb_id);
+
+            ResultSet resultSet = stmt.executeQuery();
+
+            while (resultSet.next()){
+                Comment comment = new Comment();
+                comment.setClimbId(climb_id);
+                comment.setComment(resultSet.getString("comment_text"));
+                comment.setDate(resultSet.getLong("date_text"));
```

Comment:
might be worth renaming the database column here, since it's not text at this point